src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx

```diff
@@ -11,9 +11,12 @@ import {
 import { ArtworkDetailsAdditionalInfo_artwork } from "v2/__generated__/ArtworkDetailsAdditionalInfo_artwork.graphql"
 import React, { useState } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
-
 import { RequestConditionReportQueryRenderer } from "./RequestConditionReport"
 import { ArtworkDetailsMediumModalFragmentContainer } from "../ArtworkDetailsMediumModal"
+import { useAnalyticsContext } from "v2/Artsy/Analytics/AnalyticsContext"
+import { useTracking } from "v2/Artsy"
+import { ContextModule } from "@artsy/cohesion"
+import * as Schema from "v2/Artsy/Analytics/Schema"
```

Comment:
@eessex how would you feel about a centralized index file that re-exports all the things needed to construct a `trackEvent` — every time I go to do this I'm very confused about what I'm importing from where, particularly since there's a few different imports that get picked up by Vscode (force, reaction, cohesion), and I generally don't know what's been deprecated, etc.

Reply:
The type errors also don't seem to guide me much:

![](http://static.damonzucconi.com/_capture/2d1GbuK6UaOR.png)

But using `Schema.Type.Link` passes? 😕

Reply:
Ok, re the first point there is an index already, duh, and I'll just add the analytics context exports to that.

I don't know about the type errors there but I think this can just be chalked up to my unfamiliarity with analytics stuff.